### DISCUSSION TITLE
2장 객체 생성과 파괴

### DISCUSSION THREAD

**Discussion by Jaewui** [In-houseStudyGroup/Effective-Java-Study#4] [1. 발표 정리]
[객체 생성과 파괴 발표 자료](https://www.notion.so/2238ca23f9b5405fa72c91ecb33d1700)

**Comment by 4862abd** [comment]
> [어디로 갈까요?](https://military-chill-8f9.notion.site/cb8a48480f3e48f6b885dd8b0e035f39)

퇴근하시죠

**Comment by tjdxo1193** [comment]
황성태 발표 자료 : https://foul-sense-f72.notion.site/Chapter-2-90afc63d5cac4866bdbe7e0a477ff961
